pkgs/top-level/python-packages.nix

```diff
@@ -186,6 +186,8 @@ in {
 
   asn1crypto = callPackage ../development/python-modules/asn1crypto { };
 
+  aspy.yaml = callPackage ../development/python-modules/aspy.yaml { };
```

Comment:
The `.` should be replaced by e.g. `_` (also in the commit message).

Reply:
preferable even a `-`, so to follow the PEP on normalized names.

Reply:
Yeah, I wasn't sure what to do with the period, I changed it now to `-`.